src/app/services/raiden.service.ts

```diff
@@ -184,40 +186,39 @@ export class RaidenService {
         tokenAddress: string,
         partnerAddress: string,
         settleTimeout: number,
-        balance: number,
+        balance: BigNumber,
```

Comment:
I was thinking that since we moved to `BigNumber` already it doesn't make sense anymore to actually pass the decimals all the way down from the input component to the `RaidenService`. We could do the `amountFromDecimal` conversion as close to the input as possible and remove the decimals from the service public interface signatures.